<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>729</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>729</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>833</width>
    <height>729</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>671</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="game_grid"/>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="time_bar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="game_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>      Waggle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти в главное меню</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>610</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать уровень</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_game">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>610</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string> Начать заново</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
